cmd/utils/dev.go

```diff
@@ -50,11 +52,37 @@ func LoadDev(devPath, namespace, k8sContext string) (*model.Dev, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	loadDevRc(dev)
 	loadContext(dev, k8sContext)
 	loadNamespace(dev, namespace)
 	return dev, nil
 }
 
+func loadDevRc(dev *model.Dev) {
+	defaultDevRcPath := filepath.Join(config.GetOktetoHome(), ".okteto.yml")
+	secondaryDevRcPath := filepath.Join(config.GetOktetoHome(), ".okteto.yaml")
+	var devRc *model.DevRC
+	var err error
+	if model.FileExists(defaultDevRcPath) {
+		devRc, err = model.GetRc(defaultDevRcPath)
+		if err != nil {
+			log.Infof("error while reading .dev file: %s", err.Error())
+		}
+	} else if model.FileExists(secondaryDevRcPath) {
+		devRc, err = model.GetRc(secondaryDevRcPath)
+		if err != nil {
+			log.Infof("error while reading .dev file: %s", err.Error())
+		}
+	} else {
+		log.Info("developer level dev file not found")
```

Comment:
since the default is that this file won't exist, do you think we need to log it every time? maybe only log it when it does exist?